### PR TITLE
Feature branch

### PR DESCRIPTION
**Pull request by MohamedHNoor**
Created my first Microverse Project

### CODE REVIEW

File: .github/workflow/linters.yml [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+on: pull_request
+
+env:
+  FORCE_COLOR: 1
+
+jobs:
+  lighthouse:
+    name: Lighthouse
+    runs-on: ubuntu-18.04
```
Comment (Ugboaja-Uchechi) [inline .github/workflow/linters.yml:11]: - [x] Great job setting up the `linters.yml` file 👍 . Kindly make sure you also set up  `lighthouse` `stylelint` and `webhint` by following this [article](https://github.com/microverseinc/linters-config/tree/master/html-css) provided by Microverse 🙏 . Using linters as a developer would help you learn more about writing DRY code

**Review by Ugboaja-Uchechi — CHANGES_REQUESTED** [review]
Hi @MohamedHNoor , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

- Good job with your first try ✔️ 
- README is descriptive 👍 

## Required Changes ♻️
- [x] Kindly make sure your pull request is descriptive as this would help anyone going through your GitHub understand what you did. Please add a description over here ⬇️ 
image


_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: .github/workflow/linters.yml (hunk 1 above)
Comment (MohamedHNoor) [reply .github/workflow/linters.yml:11]: Hi @Ugboaja-Uchechi  
I hope you are well.
Thank you for the review.

I added the stylelint and Webhint also provided a description for the pull request.

**Comment by Ugboaja-Uchechi** [comment]
Hi @MohamedHNoor, to request another review, you have to re-submit this pull request in your dashboard the same way you did the first time ☺️

**Review by ichala — CHANGES_REQUESTED** [review]
Hi @MohamedHNoor ,
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark: 
 
68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f6c3366516333614356304871707a5763552f67697068792e676966


Good job implementing the required changes :+1: Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**